packages/stylelint-config/package.json

```
@@ -0,0 +1,41 @@
+{
+  "name": "@ramseyinhouse/stylelint-config",
+  "version": "0.0.0",
+  "author": "Ramsey Solutions",
+  "description": "Ramsey Solutions standard stylelint configuration.",
+  "homepage": "https://github.com/ramseyinhouse/frontend/packages/stylelint-config",
+  "license": "GPL-3.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/RamseyInHouse/frontend.git"
+  },
+  "keywords": [
```

🗣️  Maybe for better npm SEO?

```suggestion
  "keywords": [
     "css", 
     "scss",
     "sass",
```